v2/datadog/aws-cloudwatch-log-events/cloudwatch_test.go

```diff
@@ -83,8 +84,8 @@ var expectedDatadogLogs = []interface{}{
 		"message":          "msg",
 		"service":          "keckebarn",
 		"timestamp":        validLambdaLogEvent.LogEvents[1].Timestamp,
-		"dd.trace_id":      uint64(123),
-		"dd.span_id":       uint64(456),
+		"dd.trace_id":      uint64(123), // nolint: gomnd
```

Comment:
Why did you not use a const trace_id = 123 here?

Reply:
don't believe it adds any value, it is just a magic number used in a test